astropy/units/tests/test_units.py

```diff
@@ -128,6 +128,18 @@ def test_units_manipulation():
     (u.AA * u.erg) ** 9
 
 
+def test_compose():
+    x =  u.m.compose(units=(u.m))
```

Comment:
I think this is sufficiently tested elsewhere, so I would instead replace it with something else that uses a tuple (but then add the trailing `,`!). Also, maybe make the test a bit more realistic by adding multiple possible units. E.g.,
```
x = (u.km/u.s).compose(units=(u.pc, u.Myr))
assert x[0].bases == [u.pc, u.Myr]
assert_allclose(x[0].scale, 1.0227121650537077)